templates/glue-jobs.yaml

```diff
@@ -14,6 +14,13 @@ Parameters:
       this is used to build up the path to the correct object keys in the bucket
     Default: main
 
+  CodeRepositoryName:
+    Type: String
+    Description: >-
+      Name of code repository, used to build up the path to the correct
+      object keys in the bucket
+    Default: BridgeDownstream
+
```

Comment:
This is added because BridgeDownstream isn't the only thing writing to the template artifacts bucket.